app/models/education_benefits_claim.rb

```diff
@@ -24,27 +13,17 @@ class EducationBenefitsClaim < ActiveRecord::Base
     chapter30
   ).freeze
 
-  validates(:form, :form_type, presence: true)
-  validates(:form_type, inclusion: FORM_TYPES)
-  validate(:form_matches_schema)
-  validate(:form_must_be_string)
+  belongs_to(:saved_claim, class_name: 'SavedClaim::EducationBenefits', inverse_of: :education_benefits_claim)
 
   has_one(:education_benefits_submission, inverse_of: :education_benefits_claim)
 
-  attr_encrypted(:form, key: Settings.db_encryption_key)
+  delegate(:parsed_form, to: :saved_claim)
+  delegate(:form, to: :saved_claim)
 
-  # initially only completed claims are allowed, later we can allow claims that dont have a submitted_at yet
-  before_validation(:set_submitted_at, on: :create)
   before_save(:set_region)
-  after_save(:create_education_benefits_submission)
+  after_create(:create_education_benefits_submission)
   after_save(:update_education_benefits_submission_status)
```

Comment:
Just want to double check that in migrations we're skipping any of these callbacks to avoid N+1